src/driver/uniformly_random_test_point_generator.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Uniformly random test point generator.
+
+Each test point is generated by choosing the spend amount for each
+publisher independently and uniformly at random from the interval
+[0, s_i], where s_i is the maximum possible spend for publisher i.
+"""
+
+import numpy as np
+from typing import Iterable
+from typing import List
+from wfa_planning_evaluation_framework.data_generators.data_set import DataSet
+from wfa_planning_evaluation_framework.driver.test_point_generator import (
+    TestPointGenerator,
+)
+
+
+class UniformlyRandomTestPointGenerator(TestPointGenerator):
+    """Generates a collection of test points for a given simulation."""
+
+    def __init__(self, dataset: DataSet, rng: np.random.Generator):
```

Comment:
Perhaps, note that
1. np.random.Generator can be replaced with np.random.RandomState (as used before)
2. Generator is more efficient than RandomState for distribution. Correct me if I'm wrong!

Reply:
np.random.RandomState is deprecated.  See below.

https://numpy.org/doc/stable/reference/random/legacy.html

Reply:
Oh, great to know!